From ff77e4cafea3b62477a13a6e14e79049c3646a9c Mon Sep 17 00:00:00 2001
From: Yann Collette <yc@example.com>
Date: Tue, 7 Apr 2026 09:49:40 +0200
Subject: [PATCH] patch

---
 src/gui/gui_manager.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/gui/gui_manager.cpp b/src/gui/gui_manager.cpp
index ace0512..76b21ee 100644
--- a/src/gui/gui_manager.cpp
+++ b/src/gui/gui_manager.cpp
@@ -20,9 +20,9 @@
 #pragma GCC diagnostic ignored "-Wmaybe-uninitialized"
 #endif
 #define NANOSVG_IMPLEMENTATION
-#include "nanosvg.h"
+#include <nanosvg/nanosvg.h>
 #define NANOSVGRAST_IMPLEMENTATION
-#include "nanosvgrast.h"
+#include <nanosvg/nanosvgrast.h>
 #pragma GCC diagnostic pop
 
 namespace Amplitron {
-- 
2.53.0

