From 44756ffbb9c9a7bc63b7cd228e370bed5ca7c30c Mon Sep 17 00:00:00 2001
From: Yann Collette <yc@example.com>
Date: Thu, 2 Apr 2026 12:51:40 +0200
Subject: [PATCH] patch

---
 src/gui/gui_manager.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/gui/gui_manager.cpp b/src/gui/gui_manager.cpp
index 5b11dc6..fbb684b 100644
--- a/src/gui/gui_manager.cpp
+++ b/src/gui/gui_manager.cpp
@@ -17,9 +17,9 @@
 #pragma GCC diagnostic push
 #pragma GCC diagnostic ignored "-Wmaybe-uninitialized"
 #define NANOSVG_IMPLEMENTATION
-#include "nanosvg.h"
+#include <nanosvg/nanosvg.h>
 #define NANOSVGRAST_IMPLEMENTATION
-#include "nanosvgrast.h"
+#include <nanosvg/nanosvgrast.h>
 #pragma GCC diagnostic pop
 
 namespace GuitarAmp {
-- 
2.53.0

