From 5d1781316025d767f3d9b25fef9042d2763174b0 Mon Sep 17 00:00:00 2001
From: Yann Collette <yc@example.com>
Date: Tue, 31 Mar 2026 22:53:11 +0200
Subject: [PATCH] fix nanosvg header

---
 src/gui/gui_manager.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/gui/gui_manager.cpp b/src/gui/gui_manager.cpp
index 405dbfd..6191bde 100644
--- a/src/gui/gui_manager.cpp
+++ b/src/gui/gui_manager.cpp
@@ -15,9 +15,9 @@
 #include <cstdio>
 
 #define NANOSVG_IMPLEMENTATION
-#include "nanosvg.h"
+#include <nanosvg/nanosvg.h>
 #define NANOSVGRAST_IMPLEMENTATION
-#include "nanosvgrast.h"
+#include <nanosvg/nanosvgrast.h>
 
 namespace GuitarAmp {
 
-- 
2.53.0

