From d2f06bf5ba1cb57a03379a752997b231f93eb070 Mon Sep 17 00:00:00 2001
From: Yann Collette <yc@example.com>
Date: Wed, 1 Apr 2026 19:29:20 +0200
Subject: [PATCH] set presets path

---
 src/preset_manager.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/preset_manager.cpp b/src/preset_manager.cpp
index e493cdd..b5bc097 100644
--- a/src/preset_manager.cpp
+++ b/src/preset_manager.cpp
@@ -29,7 +29,7 @@ namespace GuitarAmp {
 std::string PresetManager::last_error_;
 
 std::string PresetManager::get_presets_dir() {
-    std::string dir = "presets";
+    std::string dir = "/usr/share/amplitron/presets";
     MKDIR(dir.c_str());
     return dir;
 }
-- 
2.53.0

