diff --git a/src/interface.c b/src/interface.c
index f981122..977c80a 100644
--- a/src/interface.c
+++ b/src/interface.c
@@ -5066,11 +5066,11 @@ create_pref_dialog (void)
   GtkWidget *vbox163;
   GtkWidget *hbox70;
   GtkWidget *MinGainLabel;
-  GObject *MinGainSpin_adj;
+  GtkAdjustment *MinGainSpin_adj;
   GtkWidget *MinGainSpin;
   GtkWidget *hbox71;
   GtkWidget *MaxGainLabel;
-  GObject *MaxGainSpin_adj;
+  GtkAdjustment *MaxGainSpin_adj;
   GtkWidget *MaxGainSpin;
   GtkWidget *GraphicEQLabel;
   GtkWidget *hbox78;
@@ -5078,14 +5078,14 @@ create_pref_dialog (void)
   GtkWidget *CrossfadeFrame;
   GtkWidget *hbox72;
   GtkWidget *CrossfadeTimeLabel;
-  GObject *CrossfadeTimeSpin_adj;
+  GtkAdjustment *CrossfadeTimeSpin_adj;
   GtkWidget *CrossfadeTimeSpin;
   GtkWidget *CrossfadeLabel;
   GtkWidget *warningFrame;
   GtkWidget *alignment7;
   GtkWidget *warningLevelHbox;
   GtkWidget *Meter_warning_level__dbFS_;
-  GObject *warningLevelSpinButton_adj;
+  GtkAdjustment *warningLevelSpinButton_adj;
   GtkWidget *warningLevelSpinButton;
   GtkWidget *warningLabel;
   GtkWidget *SpectrumEvent;
@@ -5095,7 +5095,7 @@ create_pref_dialog (void)
   GtkWidget *SpectrumComboBox;
   GtkWidget *hbox73;
   GtkWidget *UpdateFrequencyLabel;
-  GObject *UpdateFrequencySpin_adj;
+  GtkAdjustment *UpdateFrequencySpin_adj;
   GtkWidget *UpdateFrequencySpin;
   GtkWidget *SpectrumLabel;
   GtkWidget *hbox75;
@@ -5109,7 +5109,7 @@ create_pref_dialog (void)
   GtkWidget *rmsWindowFrame;
   GtkWidget *table17;
   GtkWidget *rmsTimeLabel;
-  GObject *rmsTimeValue_adj;
+  GtkAdjustment *rmsTimeValue_adj;
   GtkWidget *rmsTimeValue;
   GtkWidget *rmsSamplesLabel;
   GtkWidget *rmsSamples;
@@ -5119,9 +5119,9 @@ create_pref_dialog (void)
   GtkWidget *DelayTable;
   GtkWidget *LowDelayLabel;
   GtkWidget *MidDelayLabel;
-  GObject *LowDelaySpinButton_adj;
+  GtkAdjustment *LowDelaySpinButton_adj;
   GtkWidget *LowDelaySpinButton;
-  GObject *MidDelaySpinButton_adj;
+  GtkAdjustment *MidDelaySpinButton_adj;
   GtkWidget *MidDelaySpinButton;
   GtkWidget *label3180;
   GtkWidget *hbox77;
@@ -5159,7 +5159,7 @@ create_pref_dialog (void)
   gtk_window_set_type_hint (GTK_WINDOW (pref_dialog), GDK_WINDOW_TYPE_HINT_DIALOG);
 
  // dialog_vbox1 = GTK_DIALOG (pref_dialog)->vbox;
-  dialog_vbox1 = gtk_dialog_get_content_area(pref_dialog); 
+  dialog_vbox1 = gtk_dialog_get_content_area(GTK_DIALOG(pref_dialog));
   gtk_widget_set_name (dialog_vbox1, "dialog_vbox1");
   gtk_widget_show (dialog_vbox1);
 
@@ -5190,7 +5190,7 @@ create_pref_dialog (void)
   gtk_widget_show (MinGainLabel);
   gtk_box_pack_start (GTK_BOX (hbox70), MinGainLabel, FALSE, FALSE, 0);
 
-  MinGainSpin_adj = gtk_adjustment_new (-12, -70, -1, 1, 10, 10);
+  MinGainSpin_adj = GTK_ADJUSTMENT(gtk_adjustment_new (-12, -70, -1, 1, 10, 10));
   MinGainSpin = gtk_spin_button_new (GTK_ADJUSTMENT (MinGainSpin_adj), 1, 0);
   gtk_widget_set_name (MinGainSpin, "MinGainSpin");
   gtk_widget_show (MinGainSpin);
@@ -5207,7 +5207,7 @@ create_pref_dialog (void)
   gtk_widget_show (MaxGainLabel);
   gtk_box_pack_start (GTK_BOX (hbox71), MaxGainLabel, FALSE, FALSE, 0);
 
-  MaxGainSpin_adj = gtk_adjustment_new (12, 1, 70, 1, 10, 10);
+  MaxGainSpin_adj = GTK_ADJUSTMENT(gtk_adjustment_new (12, 1, 70, 1, 10, 10));
   MaxGainSpin = gtk_spin_button_new (GTK_ADJUSTMENT (MaxGainSpin_adj), 1, 0);
   gtk_widget_set_name (MaxGainSpin, "MaxGainSpin");
   gtk_widget_show (MaxGainSpin);
@@ -5247,7 +5247,7 @@ create_pref_dialog (void)
   gtk_widget_show (CrossfadeTimeLabel);
   gtk_box_pack_start (GTK_BOX (hbox72), CrossfadeTimeLabel, FALSE, FALSE, 0);
 
-  CrossfadeTimeSpin_adj = gtk_adjustment_new (1, 0, 2, 0.10000000149, 0.10000000149, 0.10000000149);
+  CrossfadeTimeSpin_adj = GTK_ADJUSTMENT(gtk_adjustment_new (1, 0, 2, 0.10000000149, 0.10000000149, 0.10000000149));
   CrossfadeTimeSpin = gtk_spin_button_new (GTK_ADJUSTMENT (CrossfadeTimeSpin_adj), 1, 1);
   gtk_widget_set_name (CrossfadeTimeSpin, "CrossfadeTimeSpin");
   gtk_widget_show (CrossfadeTimeSpin);
@@ -5322,10 +5322,10 @@ create_pref_dialog (void)
   gtk_widget_set_name (SpectrumComboBox, "SpectrumComboBox");
   gtk_widget_show (SpectrumComboBox);
   gtk_container_add (GTK_CONTAINER (eventbox69), SpectrumComboBox);
-  gtk_combo_box_text_append (GTK_COMBO_BOX (SpectrumComboBox), NULL, _("Pre EQ"));
-  gtk_combo_box_text_append (GTK_COMBO_BOX (SpectrumComboBox), NULL, _("Post EQ"));
-  gtk_combo_box_text_append (GTK_COMBO_BOX (SpectrumComboBox), NULL, _("Post Compressor"));
-  gtk_combo_box_text_append (GTK_COMBO_BOX (SpectrumComboBox), NULL,_("Output"));
+  gtk_combo_box_text_append (GTK_COMBO_BOX_TEXT (SpectrumComboBox), NULL, _("Pre EQ"));
+  gtk_combo_box_text_append (GTK_COMBO_BOX_TEXT (SpectrumComboBox), NULL, _("Post EQ"));
+  gtk_combo_box_text_append (GTK_COMBO_BOX_TEXT (SpectrumComboBox), NULL, _("Post Compressor"));
+  gtk_combo_box_text_append (GTK_COMBO_BOX_TEXT (SpectrumComboBox), NULL,_("Output"));
 
   hbox73 = gtk_hbox_new (TRUE, 0);
   gtk_widget_set_name (hbox73, "hbox73");
@@ -5409,7 +5409,7 @@ create_pref_dialog (void)
   rmsTimeLabel = gtk_label_new (_("Time (ms):"));
   gtk_widget_set_name (rmsTimeLabel, "rmsTimeLabel");
   gtk_widget_show (rmsTimeLabel);
-  gtk_grid_attach (GTK_GRID (table17), rmsTimeLabel, 0, 1, 0, 1);
+  gtk_grid_attach (GTK_GRID (table17), rmsTimeLabel, 0, 1, 1, 1);
    //                 (GtkAttachOptions) (GTK_FILL),
    //                 (GtkAttachOptions) (0), 0, 0);
   gtk_misc_set_alignment (GTK_MISC (rmsTimeLabel), 0, 0.5);
@@ -5418,7 +5418,7 @@ create_pref_dialog (void)
   rmsTimeValue = gtk_spin_button_new (GTK_ADJUSTMENT (rmsTimeValue_adj), 1, 0);
   gtk_widget_set_name (rmsTimeValue, "rmsTimeValue");
   gtk_widget_show (rmsTimeValue);
-  gtk_grid_attach (GTK_GRID (table17), rmsTimeValue, 1, 2, 0, 1);
+  gtk_grid_attach (GTK_GRID (table17), rmsTimeValue, 1, 2, 1, 1);
   //                  (GtkAttachOptions) (GTK_EXPAND | GTK_FILL),
    //                 (GtkAttachOptions) (0), 0, 0);
 
@@ -5465,7 +5465,7 @@ create_pref_dialog (void)
   LowDelayLabel = gtk_label_new (_("Low band"));
   gtk_widget_set_name (LowDelayLabel, "LowDelayLabel");
   gtk_widget_show (LowDelayLabel);
-  gtk_grid_attach (GTK_GRID (DelayTable), LowDelayLabel, 0, 1, 0, 1);
+  gtk_grid_attach (GTK_GRID (DelayTable), LowDelayLabel, 0, 1, 1, 1);
   //                  (GtkAttachOptions) (GTK_FILL),
   //                  (GtkAttachOptions) (0), 0, 0);
   gtk_misc_set_alignment (GTK_MISC (LowDelayLabel), 0, 0.5);
@@ -5478,16 +5478,16 @@ create_pref_dialog (void)
   //                  (GtkAttachOptions) (0), 0, 0);
   gtk_misc_set_alignment (GTK_MISC (MidDelayLabel), 0, 0.5);
 
-  LowDelaySpinButton_adj = gtk_adjustment_new (2, 0, 2, 0.00999999977648, 0.10000000149, 0.10000000149);
+  LowDelaySpinButton_adj = GTK_ADJUSTMENT(gtk_adjustment_new (2, 0, 2, 0.00999999977648, 0.10000000149, 0.10000000149));
   LowDelaySpinButton = gtk_spin_button_new (GTK_ADJUSTMENT (LowDelaySpinButton_adj), 1, 2);
   gtk_widget_set_name (LowDelaySpinButton, "LowDelaySpinButton");
   gtk_widget_show (LowDelaySpinButton);
-  gtk_grid_attach (GTK_GRID (DelayTable), LowDelaySpinButton, 1, 2, 0, 1);
+  gtk_grid_attach (GTK_GRID (DelayTable), LowDelaySpinButton, 1, 2, 1, 1);
   //                  (GtkAttachOptions) (GTK_EXPAND | GTK_FILL),
   //                  (GtkAttachOptions) (0), 0, 0);
   gtk_spin_button_set_numeric (GTK_SPIN_BUTTON (LowDelaySpinButton), TRUE);
 
-  MidDelaySpinButton_adj = gtk_adjustment_new (0.5, 0, 0.5, 0.00999999977648, 0.10000000149, 0.10000000149);
+  MidDelaySpinButton_adj = GTK_ADJUSTMENT(gtk_adjustment_new (0.5, 0, 0.5, 0.00999999977648, 0.10000000149, 0.10000000149));
   MidDelaySpinButton = gtk_spin_button_new (GTK_ADJUSTMENT (MidDelaySpinButton_adj), 1, 2);
   gtk_widget_set_name (MidDelaySpinButton, "MidDelaySpinButton");
   gtk_widget_show (MidDelaySpinButton);
@@ -5528,21 +5528,21 @@ create_pref_dialog (void)
   gtk_widget_set_name (ColorsComboBox, "ColorsComboBox");
   gtk_widget_show (ColorsComboBox);
   gtk_container_add (GTK_CONTAINER (eventbox68), ColorsComboBox);
-  gtk_combo_box_text_append (GTK_COMBO_BOX (ColorsComboBox), NULL,_("Low Band Compressor"));
-  gtk_combo_box_text_append (GTK_COMBO_BOX (ColorsComboBox), NULL,_("Mid Band Compressor"));
-  gtk_combo_box_text_append (GTK_COMBO_BOX (ColorsComboBox), NULL,_("High Band Compressor"));
-  gtk_combo_box_text_append (GTK_COMBO_BOX (ColorsComboBox), NULL,_("Ganged Controls"));
-  gtk_combo_box_text_append (GTK_COMBO_BOX (ColorsComboBox), NULL,_("Parametric Handles"));
-  gtk_combo_box_text_append (GTK_COMBO_BOX (ColorsComboBox), NULL,_("HDEQ Curve"));
-  gtk_combo_box_text_append (GTK_COMBO_BOX (ColorsComboBox), NULL,_("HDEQ Spectrum"));
-  gtk_combo_box_text_append (GTK_COMBO_BOX (ColorsComboBox), NULL,_("HDEQ Grid"));
-  gtk_combo_box_text_append (GTK_COMBO_BOX (ColorsComboBox), NULL,_("HDEQ Background"));
-  gtk_combo_box_text_append (GTK_COMBO_BOX (ColorsComboBox), NULL,_("Text"));
-  gtk_combo_box_text_append (GTK_COMBO_BOX (ColorsComboBox), NULL,_("Meter Normal"));
-  gtk_combo_box_text_append (GTK_COMBO_BOX (ColorsComboBox), NULL, _("Meter Warning"));
-  gtk_combo_box_text_append (GTK_COMBO_BOX (ColorsComboBox), NULL,_("Meter Over"));
-  gtk_combo_box_text_append (GTK_COMBO_BOX (ColorsComboBox), NULL,_("Meter Peak"));
-  gtk_combo_box_text_append (GTK_COMBO_BOX (ColorsComboBox), NULL,_("Reset All Colors To Defaults"));
+  gtk_combo_box_text_append (GTK_COMBO_BOX_TEXT (ColorsComboBox), NULL,_("Low Band Compressor"));
+  gtk_combo_box_text_append (GTK_COMBO_BOX_TEXT (ColorsComboBox), NULL,_("Mid Band Compressor"));
+  gtk_combo_box_text_append (GTK_COMBO_BOX_TEXT (ColorsComboBox), NULL,_("High Band Compressor"));
+  gtk_combo_box_text_append (GTK_COMBO_BOX_TEXT (ColorsComboBox), NULL,_("Ganged Controls"));
+  gtk_combo_box_text_append (GTK_COMBO_BOX_TEXT (ColorsComboBox), NULL,_("Parametric Handles"));
+  gtk_combo_box_text_append (GTK_COMBO_BOX_TEXT (ColorsComboBox), NULL,_("HDEQ Curve"));
+  gtk_combo_box_text_append (GTK_COMBO_BOX_TEXT (ColorsComboBox), NULL,_("HDEQ Spectrum"));
+  gtk_combo_box_text_append (GTK_COMBO_BOX_TEXT (ColorsComboBox), NULL,_("HDEQ Grid"));
+  gtk_combo_box_text_append (GTK_COMBO_BOX_TEXT (ColorsComboBox), NULL,_("HDEQ Background"));
+  gtk_combo_box_text_append (GTK_COMBO_BOX_TEXT (ColorsComboBox), NULL,_("Text"));
+  gtk_combo_box_text_append (GTK_COMBO_BOX_TEXT (ColorsComboBox), NULL,_("Meter Normal"));
+  gtk_combo_box_text_append (GTK_COMBO_BOX_TEXT (ColorsComboBox), NULL, _("Meter Warning"));
+  gtk_combo_box_text_append (GTK_COMBO_BOX_TEXT (ColorsComboBox), NULL,_("Meter Over"));
+  gtk_combo_box_text_append (GTK_COMBO_BOX_TEXT (ColorsComboBox), NULL,_("Meter Peak"));
+  gtk_combo_box_text_append (GTK_COMBO_BOX_TEXT (ColorsComboBox), NULL,_("Reset All Colors To Defaults"));
 
   ColorsLabel = gtk_label_new (_("<i><b>Colors</b></i>"));
   gtk_widget_set_name (ColorsLabel, "ColorsLabel");
@@ -5572,8 +5572,8 @@ create_pref_dialog (void)
   gtk_widget_set_name (limiter_combo, "limiter_combo");
   gtk_widget_show (limiter_combo);
   gtk_container_add (GTK_CONTAINER (eventbox67), limiter_combo);
-  gtk_combo_box_text_append (GTK_COMBO_BOX (limiter_combo), NULL,_("Steve Harris' fast_lookahead_limiter"));
-  gtk_combo_box_text_append (GTK_COMBO_BOX (limiter_combo), NULL,_("Sampo Savolainen's foo_limiter"));
+  gtk_combo_box_text_append (GTK_COMBO_BOX_TEXT (limiter_combo), NULL,_("Steve Harris' fast_lookahead_limiter"));
+  gtk_combo_box_text_append (GTK_COMBO_BOX_TEXT (limiter_combo), NULL,_("Sampo Savolainen's foo_limiter"));
 
   limiter_frame_label = gtk_label_new (_("<b>Limiter</b>"));
   gtk_widget_set_name (limiter_frame_label, "limiter_frame_label");
@@ -5632,7 +5632,7 @@ create_pref_dialog (void)
   gtk_frame_set_label_widget (GTK_FRAME (output_meter_numeric_display_frame), output_meter_numeric_display_label);
   gtk_label_set_use_markup (GTK_LABEL (output_meter_numeric_display_label), TRUE);
 
-  dialog_action_area1 = gtk_dialog_get_action_area(pref_dialog); // GTK_DIALOG (pref_dialog)->action_area;
+  dialog_action_area1 = gtk_dialog_get_action_area(GTK_DIALOG(pref_dialog)); // GTK_DIALOG (pref_dialog)->action_area;
   gtk_widget_set_name (dialog_action_area1, "dialog_action_area1");
   gtk_widget_show (dialog_action_area1);
   gtk_button_box_set_layout (GTK_BUTTON_BOX (dialog_action_area1), GTK_BUTTONBOX_EDGE);
@@ -5909,7 +5909,7 @@ create_scene_name_dialog (void)
   gtk_window_set_title (GTK_WINDOW (scene_name_dialog), _("Scene X Name"));
   gtk_window_set_type_hint (GTK_WINDOW (scene_name_dialog), GDK_WINDOW_TYPE_HINT_DIALOG);
 
-  dialog_vbox2 = gtk_dialog_get_content_area(scene_name_dialog);
+  dialog_vbox2 = gtk_dialog_get_content_area(GTK_DIALOG(scene_name_dialog));
   gtk_widget_set_name (dialog_vbox2, "dialog_vbox2");
   gtk_widget_show (dialog_vbox2);
 
@@ -5919,7 +5919,7 @@ create_scene_name_dialog (void)
   gtk_box_pack_start (GTK_BOX (dialog_vbox2), scene_name_entry, FALSE, FALSE, 0);
   gtk_entry_set_max_length (GTK_ENTRY (scene_name_entry), 99);
 
-  dialog_action_area3 = gtk_dialog_get_action_area(scene_name_dialog);
+  dialog_action_area3 = gtk_dialog_get_action_area(GTK_DIALOG(scene_name_dialog));
   gtk_widget_set_name (dialog_action_area3, "dialog_action_area3");
   gtk_widget_show (dialog_action_area3);
   gtk_button_box_set_layout (GTK_BUTTON_BOX (dialog_action_area3), GTK_BUTTONBOX_END);
@@ -6190,7 +6190,7 @@ create_about_dialog (void)
   gtk_window_set_title (GTK_WINDOW (about_dialog), _("About JAMin"));
   gtk_window_set_type_hint (GTK_WINDOW (about_dialog), GDK_WINDOW_TYPE_HINT_DIALOG);
 
-  dialog_vbox3 = gtk_dialog_get_content_area(about_dialog);
+  dialog_vbox3 = gtk_dialog_get_content_area(GTK_DIALOG(about_dialog));
   gtk_widget_set_name (dialog_vbox3, "dialog_vbox3");
   gtk_widget_show (dialog_vbox3);
   gtk_widget_set_size_request (dialog_vbox3, 395, 431);
@@ -6300,7 +6300,7 @@ create_about_dialog (void)
   gtk_widget_show (label330);
   gtk_notebook_set_tab_label (GTK_NOTEBOOK (notebook2), gtk_notebook_get_nth_page (GTK_NOTEBOOK (notebook2), 3), label330);
 
-  dialog_action_area4 = gtk_dialog_get_action_area(about_dialog);
+  dialog_action_area4 = gtk_dialog_get_action_area(GTK_DIALOG(about_dialog));
   gtk_widget_set_name (dialog_action_area4, "dialog_action_area4");
   gtk_widget_show (dialog_action_area4);
   gtk_button_box_set_layout (GTK_BUTTON_BOX (dialog_action_area4), GTK_BUTTONBOX_END);
@@ -6606,13 +6606,13 @@ create_window3 (void)
   gtk_widget_set_name (combobox1, "combobox1");
   gtk_widget_show (combobox1);
   gtk_box_pack_start (GTK_BOX (vbox167), combobox1, FALSE, TRUE, 0);
-  gtk_combo_box_text_append (GTK_COMBO_BOX (combobox1), NULL, _("Rock"));
-  gtk_combo_box_text_append (GTK_COMBO_BOX (combobox1), NULL, _("Classical"));
-  gtk_combo_box_text_append (GTK_COMBO_BOX (combobox1), NULL, _("Jazz"));
-  gtk_combo_box_text_append (GTK_COMBO_BOX (combobox1), NULL, _("Blues"));
-  gtk_combo_box_text_append (GTK_COMBO_BOX (combobox1), NULL, _("Techno"));
-  gtk_combo_box_text_append (GTK_COMBO_BOX (combobox1), NULL, _("DnB"));
-  gtk_combo_box_text_append (GTK_COMBO_BOX (combobox1), NULL, _("Pop"));
+  gtk_combo_box_text_append (GTK_COMBO_BOX_TEXT (combobox1), NULL, _("Rock"));
+  gtk_combo_box_text_append (GTK_COMBO_BOX_TEXT (combobox1), NULL, _("Classical"));
+  gtk_combo_box_text_append (GTK_COMBO_BOX_TEXT (combobox1), NULL, _("Jazz"));
+  gtk_combo_box_text_append (GTK_COMBO_BOX_TEXT (combobox1), NULL, _("Blues"));
+  gtk_combo_box_text_append (GTK_COMBO_BOX_TEXT (combobox1), NULL, _("Techno"));
+  gtk_combo_box_text_append (GTK_COMBO_BOX_TEXT (combobox1), NULL, _("DnB"));
+  gtk_combo_box_text_append (GTK_COMBO_BOX_TEXT (combobox1), NULL, _("Pop"));
 
   hbox_w3_1 = gtk_hbox_new (FALSE, 0);
   gtk_widget_set_name (hbox_w3_1, "hbox_w3_1");
