From a377bc13d52e7584b12cca5019d2b00e2c3386ea Mon Sep 17 00:00:00 2001
From: Yann Collette <ycollette.nospam@free.fr>
Date: Sun, 18 May 2025 13:28:12 +0200
Subject: [PATCH] fix config

---
 config.mk | 14 +-------------
 1 file changed, 1 insertion(+), 13 deletions(-)

diff --git a/config.mk b/config.mk
index 211c838..8ae0d5e 100644
--- a/config.mk
+++ b/config.mk
@@ -3,8 +3,6 @@ VERSION=0.234.3
 
 ARCH:=$(shell uname -m)
 
-CXXFLAGS = -Wall -Wextra -Wdeprecated-declarations -Wno-psabi -std=c++17 -pthread	\
--ggdb -fno-finite-math-only -Wno-psabi
 # -Wconversion
 # -Werror
 
@@ -15,22 +13,13 @@ UNAME_S := $(shell uname -s)
 ifeq ($(UNAME_S),Linux)
   DLLEXT=.so
   EXTERNALS +=  alsa
-  CXXFLAGS += -fext-numeric-literals
 else
   DLLEXT=.dylib
 endif
 endif
 
-
-ifeq "$(ARCH)" "x86_64"
-  CXXFLAGS += -msse -msse2 -mfpmath=sse
-  ifneq "$(UNAME_S)" "Darwin"
-    CXXFLAGS += -ffast-math
-  endif
-endif
-
 CPPFLAGS = -std=c++17
-PREFIX = /usr/local
+PREFIX = /usr/
 
 GITMODIFIED:=$(shell test -z "`git status --porcelain -uno`" || echo "-modified")
 COMMITHASH:=$(shell git log -1 --abbrev=7 --pretty='format:%h')
@@ -59,7 +48,6 @@ SOURCE_DIR = src
 export VERSION
 export SOURCE_DIR
 export BUILD_DIR
-export CXXFLAGS
 export HAS_LSL
 export HAS_OPENMHA
 export HAS_OPENCV2
-- 
2.49.0

