From e0432c70e59b8d97049dea376728ac5b00331f9c Mon Sep 17 00:00:00 2001
From: Yann Collette <ycollette.nospam@free.fr>
Date: Fri, 18 Oct 2024 21:28:30 +0200
Subject: [PATCH] fix config.mk

---
 config.mk | 13 +------------
 1 file changed, 1 insertion(+), 12 deletions(-)

diff --git a/config.mk b/config.mk
index 578dc81..bce694e 100644
--- a/config.mk
+++ b/config.mk
@@ -3,8 +3,6 @@ VERSION=0.233.1
 
 ARCH=$(shell uname -m)
 
-CXXFLAGS = -Wall -Wextra -Wdeprecated-declarations -Wno-psabi -std=c++17 -pthread	\
--ggdb -fno-finite-math-only -Wno-psabi
 # -Wconversion
 # -Werror
 
@@ -15,22 +13,14 @@ UNAME_S := $(shell uname -s)
 ifeq ($(UNAME_S),Linux)
   DLLEXT=.so
   EXTERNALS +=  alsa
-  CXXFLAGS += -fext-numeric-literals
 else
   DLLEXT=.dylib
 endif
 endif
 
 
-ifeq "$(ARCH)" "x86_64"
-  CXXFLAGS += -msse -msse2 -mfpmath=sse
-  ifneq "$(UNAME_S)" "Darwin"
-    CXXFLAGS += -ffast-math
-  endif
-endif
-
 CPPFLAGS = -std=c++17
-PREFIX = /usr/local
+PREFIX = /usr/
 
 GITMODIFIED=$(shell test -z "`git status --porcelain -uno`" || echo "-modified")
 COMMITHASH=$(shell git log -1 --abbrev=7 --pretty='format:%h')
@@ -58,7 +48,6 @@ SOURCE_DIR = src
 export VERSION
 export SOURCE_DIR
 export BUILD_DIR
-export CXXFLAGS
 export HAS_LSL
 export HAS_OPENMHA
 export HAS_OPENCV2
-- 
2.47.0

