From 9c22648e88bd3a448883a91ef4759ddd7a66d222 Mon Sep 17 00:00:00 2001
From: Yann Collette <ycollette.nospam@free.fr>
Date: Sun, 11 May 2025 15:16:36 +0200
Subject: [PATCH] fix syntax error

---
 src/dsp/src/geonkick.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/dsp/src/geonkick.c b/src/dsp/src/geonkick.c
index cc111d3..1b6de67 100755
--- a/src/dsp/src/geonkick.c
+++ b/src/dsp/src/geonkick.c
@@ -1570,7 +1570,7 @@ geonkick_ged_forced_midi_channel(struct geonkick *kick,
 }
 
 size_t
-geonkick_midi_channels_number(struct geonkick *kick)
+geonkick_midi_channels_number()
 {
         return GEONKICK_MAX_MIDI_CHANNELS;
 }
-- 
2.49.0

