From 3bcade056ed113b08db5f22c43d7f5572249b877 Mon Sep 17 00:00:00 2001
From: Yann Collette <ycollette.nospam@free.fr>
Date: Sun, 19 Mar 2023 22:08:57 +0100
Subject: [PATCH] Makefile

---
 Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index 1831117..7bcace0 100644
--- a/Makefile
+++ b/Makefile
@@ -1,11 +1,11 @@
 .POSIX:
 CC=cc
-CFLAGS_COMMON=-std=c99 -W -Wall -I.
+CFLAGS_COMMON=-std=c99 -W -Wall -I. $(DEPFLAGS)
 CFLAGS=$(CFLAGS_COMMON) -O2
 CFLAGS_FAST=$(CFLAGS_COMMON) -O3
 CFLAGS_FASTF=$(CFLAGS_COMMON) -O3 -ffast-math
 CFLAGS_SIZE=$(CFLAGS_COMMON) -Os
-LFLAGS=-s -Lsau -lsau -lm
+LFLAGS=-Lsau -lsau -lm
 LFLAGS_LINUX=$(LFLAGS) -lasound
 LFLAGS_SNDIO=$(LFLAGS) -lsndio
 LFLAGS_OSSAUDIO=$(LFLAGS) -lossaudio
-- 
2.39.2

