From 1ef41527899a7fafe9dbf8f91bb8ca4f40ed50f5 Mon Sep 17 00:00:00 2001
From: Yann Collette <ycollette.nospam@free.fr>
Date: Wed, 18 Aug 2021 06:36:01 +0200
Subject: [PATCH] soundbank in share

---
 Source/GlobalIncludes.h | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/Source/GlobalIncludes.h b/Source/GlobalIncludes.h
index a7b0bd6..91b0aa5 100644
--- a/Source/GlobalIncludes.h
+++ b/Source/GlobalIncludes.h
@@ -75,10 +75,7 @@
 #define MATRIX_SECTION_INDEX_MATRIX 0
 
 #ifdef ODIN_LINUX
-#define ODIN_STORAGE_PATH                                                                                              \
-	File::getSpecialLocation(File::SpecialLocationType::userHomeDirectory).getFullPathName() +                         \
-	    File::getSeparatorString() + ".local" + File::getSeparatorString() + "share" + File::getSeparatorString() +    \
-	    "odin2"
+#define ODIN_STORAGE_PATH "/usr/share/odin2"
 #endif
 #ifdef ODIN_MAC
 #define ODIN_STORAGE_PATH                                                                                              \
-- 
2.31.1

