From c7b89aa65dee05f01456706fbb5f56be5cbf7fb8 Mon Sep 17 00:00:00 2001
From: Yann Collette <ycollette.nospam@free.fr>
Date: Mon, 15 Sep 2025 22:05:55 +0200
Subject: [PATCH] patch

---
 extras/AudioPluginHost/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/extras/AudioPluginHost/CMakeLists.txt b/extras/AudioPluginHost/CMakeLists.txt
index 6f4144d..85ce8fa 100644
--- a/extras/AudioPluginHost/CMakeLists.txt
+++ b/extras/AudioPluginHost/CMakeLists.txt
@@ -60,7 +60,7 @@ target_compile_definitions(AudioPluginHost PRIVATE
     JUCE_CONTENT_SHARING=1
     JUCE_DIRECTSOUND=1
     JUCE_DISABLE_CAUTIOUS_PARAMETER_ID_CHECKING=1
-    JUCE_PLUGINHOST_LADSPA=1
+    JUCE_PLUGINHOST_LADSPA==${JUCE_PLUGINHOST_LADSPA}
     JUCE_PLUGINHOST_LV2=1
     JUCE_PLUGINHOST_VST3=1
     JUCE_PLUGINHOST_VST=0
-- 
2.51.0

