From f90a8254bbce84b8b9bf06c71129e7ff39757c33 Mon Sep 17 00:00:00 2001
From: Yann Collette <yc@example.com>
Date: Wed, 15 Jul 2026 21:13:10 +0200
Subject: [PATCH] fix path

---
 hi_core/hi_core/HiseSettings.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/hi_core/hi_core/HiseSettings.cpp b/hi_core/hi_core/HiseSettings.cpp
index 226095b..262a8b8 100644
--- a/hi_core/hi_core/HiseSettings.cpp
+++ b/hi_core/hi_core/HiseSettings.cpp
@@ -1336,6 +1336,7 @@ var HiseSettings::Data::getDefaultSetting(const Identifier& id) const
 	else if (id == Scripting::EnableMousePositioning) return "Yes";
 	else if (id == Scripting::CompileTimeout)		return 5.0;
 	else if (id == Scripting::SaveConnectedFilesOnCompile) return "No";
+	else if (id == Compiler::HisePath)              return "/usr/src/HISE";
 	else if (id == Scripting::RestApiPort)			return 1900;
 	else if (id == Scripting::AutoStartRestServer)	return "No";
 	else if (id == Scripting::CorsAllowedOrigins)	return "*";
-- 
2.55.0

