From 99d29ddb2937d51e8adb98b2090a1d64725763e1 Mon Sep 17 00:00:00 2001
From: Yann Collette <ycollette.nospam@free.fr>
Date: Sat, 5 Oct 2024 21:50:23 +0200
Subject: [PATCH] fix link

---
 CMakeLists.txt | 21 +--------------------
 1 file changed, 1 insertion(+), 20 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index b390e70..8d2fe05 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -56,26 +56,10 @@ include_directories(
 
 
 # Compiler options
-add_definitions(-fPIC)
-add_definitions(-Os)
-add_definitions(-O2)
-add_definitions(-Wall)
 add_definitions(
     -DGLFW_EXPOSE_NATIVE_X11
 )
 
-
-# Linker options
-set(LINK_OPTIONS -static-libstdc++)
-set(LINK_OPTIONS ${LINK_OPTIONS} -static-libgcc)
-set(LINK_OPTIONS ${LINK_OPTIONS} -Wl,--no-undefined)
-set(LINK_OPTIONS ${LINK_OPTIONS} -Wl,-Bsymbolic)
-set(LINK_OPTIONS ${LINK_OPTIONS} -Wl,-Bstatic)
-set(LINK_OPTIONS ${LINK_OPTIONS} -lfftw3f)
-set(LINK_OPTIONS ${LINK_OPTIONS} -Wl,-Bdynamic)
-set(LINK_OPTIONS ${LINK_OPTIONS} -fPIC)
-
-
 if(WIN32)
     # Resource options
 endif(WIN32)
@@ -115,10 +99,6 @@ set ( CXX_SRCS
     ${CMAKE_CURRENT_LIST_DIR}/pluginterfaces/vst2.x/audioeffectx.cpp
 )
 
-set_source_files_properties(
-    ${CXX_SRCS} PROPERTIES COMPILE_FLAGS 
-    " -fPIC -Os -O2 -Wall")
-
 if(WIN32)
     enable_language(RC)
     set(CMAKE_RC_COMPILE_OBJECT
@@ -140,6 +120,7 @@ target_link_libraries(mx_comp
     X11
     dl
     pthread
+    fftw3f
 )
 
 
-- 
2.46.2

