From 1057c3479bdf01cb13f152cf1481572bf723942c Mon Sep 17 00:00:00 2001
From: Yann Collette <ycollette.nospam@free.fr>
Date: Tue, 27 May 2025 07:16:50 +0200
Subject: [PATCH] fix cmake

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index b133f51..b93324e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -16,7 +16,7 @@ add_subdirectory(modules/JUCE)
 
 foreach(module_name IN ITEMS gin gin_dsp gin_graphics gin_gui gin_plugin gin_simd)
 	juce_add_module (
-		"${CMAKE_CURRENT_LIST_DIR}/modules/gin/modules/${module_name}"
+		"${CMAKE_CURRENT_LIST_DIR}/modules/Gin/modules/${module_name}"
 		)
 
 	set_property (TARGET "${module_name}" APPEND PROPERTY LABELS Gin)
-- 
2.49.0

