From 34b9e71eb4aba363a44f2cf718284b4ece7441a6 Mon Sep 17 00:00:00 2001
From: Yann Collette <yc@example.com>
Date: Wed, 8 Apr 2026 21:42:52 +0200
Subject: [PATCH] patch

---
 CMakeLists.txt | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 4150306..fd868a4 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -178,11 +178,6 @@ target_link_libraries(pluginval PRIVATE
     juce::juce_recommended_warning_flags
     magic_enum)
 
-if (${CMAKE_SYSTEM_NAME} MATCHES "Linux")
-    target_link_libraries(pluginval PRIVATE
-        -static-libstdc++)
-endif()
-
 # Embed the SDK's validator binary into pluginval
 if(PLUGINVAL_VST3_VALIDATOR)
     set(VSTVALIDATOR_HEADER "${CMAKE_BINARY_DIR}/generated/vstvalidator_data.h")
-- 
2.53.0

