From 7eb91ac372123d24c2174dd47ac8797149feff59 Mon Sep 17 00:00:00 2001
From: Yann Collette <ycollette.nospam@free.fr>
Date: Sun, 11 May 2025 14:14:37 +0200
Subject: [PATCH] fix syntax errors

---
 source/portmidi/pm_linux/pmlinuxalsa.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/source/portmidi/pm_linux/pmlinuxalsa.c b/source/portmidi/pm_linux/pmlinuxalsa.c
index 8e85cfe..454291f 100644
--- a/source/portmidi/pm_linux/pmlinuxalsa.c
+++ b/source/portmidi/pm_linux/pmlinuxalsa.c
@@ -201,7 +201,7 @@ static PmError alsa_write_byte(PmInternal *midi, unsigned char byte,
             /* compute relative time of event = timestamp - now + latency */
             PmTimestamp now = (midi->time_proc ? 
                                midi->time_proc(midi->time_info) : 
-                               Pt_Time(NULL));
+                               Pt_Time());
             int when = timestamp;
             /* if timestamp is zero, send immediately */
             /* otherwise compute time delay and use delay if positive */
-- 
2.49.0

