From 43778fa5f22a0853a6c15a8c82a4ae7cedd3a11e Mon Sep 17 00:00:00 2001
From: Yann Collette <ycollette.nospam@free.fr>
Date: Sat, 31 Aug 2024 19:49:25 +0200
Subject: [PATCH] add missing header

---
 source/xtJucePlugin/xtArp.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/source/xtJucePlugin/xtArp.h b/source/xtJucePlugin/xtArp.h
index 8d6202c..131b555 100644
--- a/source/xtJucePlugin/xtArp.h
+++ b/source/xtJucePlugin/xtArp.h
@@ -1,4 +1,7 @@
 #pragma once
+
+#include <cstdint>
+
 #include "jucePluginLib/event.h"
 
 namespace juce
-- 
2.46.0

