diff --git StructureSynth/GUI/MainWindow.cpp StructureSynth/GUI/MainWindow.cpp
index f800a53..51d123b 100644
--- StructureSynth/GUI/MainWindow.cpp
+++ StructureSynth/GUI/MainWindow.cpp
@@ -1075,13 +1075,13 @@ namespace StructureSynth {
 		// Mac needs to step two directies up, when debugging in XCode...
 		QString MainWindow::getExamplesDir() {
 			QStringList examplesDir;
-			examplesDir << "Examples" << "../../Examples";
+			examplesDir << DATADIR "/Examples" << "Examples";
 			return findDirectory(examplesDir);
 		}
 
 		QString MainWindow::getMiscDir() {
 			QStringList miscDir;
-			miscDir << "Misc" << "../../Misc";
+			miscDir << DATADIR "/Misc" << "Misc";
 			return findDirectory(miscDir);
 		}
 
