From 38d645d66ec7e343f7c55cfd2227ec9e4b0cfb50 Mon Sep 17 00:00:00 2001
From: Yann Collette <ycollette.nospam@free.fr>
Date: Thu, 7 Oct 2021 22:55:40 +0200
Subject: [PATCH] fix move operator

---
 Source/PluginEditor.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Source/PluginEditor.cpp b/Source/PluginEditor.cpp
index 2baf962..51ad9c8 100644
--- a/Source/PluginEditor.cpp
+++ b/Source/PluginEditor.cpp
@@ -255,7 +255,7 @@ void PaulstretchpluginAudioProcessorEditor::showRenderDialog()
 	auto contentraw =  new RenderSettingsComponent(&processor);
 	contentraw->setSize(contentraw->getWidth(), contentraw->getPreferredHeight());
 	std::unique_ptr<Component> content(contentraw);
-	CallOutBox::launchAsynchronously(std::move(content), m_render_button.getBounds(), this);
+	CallOutBox::launchAsynchronously(std::move(content.get()), m_render_button.getBounds(), this);
 }
 
 void PaulstretchpluginAudioProcessorEditor::executeModalMenuAction(int menuid, int r)
-- 
2.31.1

