From fbfc7a7e04072a687a6599867e427cec1fb1b0cf Mon Sep 17 00:00:00 2001
From: Yann Collette <ycollette.nospam@free.fr>
Date: Sat, 31 May 2025 20:38:20 +0200
Subject: [PATCH] aarch

---
 Source/aeolus/simd.cpp | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/Source/aeolus/simd.cpp b/Source/aeolus/simd.cpp
index f809249..d4e1c40 100644
--- a/Source/aeolus/simd.cpp
+++ b/Source/aeolus/simd.cpp
@@ -40,6 +40,13 @@
 #   define SIMD_AVX 1
 #endif
 
+#define SIMDE_ENABLE_NATIVE_ALIASES
+#include <simde/x86/sse3.h>
+#include <simde/x86/avx.h>
+#include <simde/x86/avx2.h>
+#include <simde/x86/avx512.h>
+#define SIMD
+
 AEOLUS_NAMESPACE_BEGIN
 
 // perform cpuid function
-- 
2.49.0

