From 6e7c6a628d9e77f4780b69ff5429312305d099e7 Mon Sep 17 00:00:00 2001
From: Yann Collette <ycollette.nospam@free.fr>
Date: Sun, 22 Sep 2024 20:39:31 +0200
Subject: [PATCH] single-lib-path

---
 data/carla-single | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/data/carla-single b/data/carla-single
index d5bfdca..73b319c 100755
--- a/data/carla-single
+++ b/data/carla-single
@@ -25,9 +25,9 @@ import sys
 # --------------------------------------------------------------------------------------------------------
 # Setup
 
-INSTALL_PREFIX = "X-PREFIX-X"
+INSTALL_PREFIX = "/usr"
 
-CARLA_LIBDIR = os.path.join(INSTALL_PREFIX, "lib", "carla")
+CARLA_LIBDIR = os.path.join(INSTALL_PREFIX, "lib64", "carla")
 CARLA_RESDIR = os.path.join(INSTALL_PREFIX, "share", "carla", "resources")
 
 # --------------------------------------------------------------------------------------------------------
-- 
2.46.1

