--- source/frontend/carla_database.py.orig	2022-03-19 16:55:20.796092166 +0100
+++ source/frontend/carla_database.py	2022-03-19 16:56:17.009719997 +0100
@@ -1358,7 +1358,7 @@
 
     # -----------------------------------------------------------------------------------------------------------------
 
-    @pyqtSlot(int, str)
+    @pyqtSlot(float, str)
     def slot_handlePluginLook(self, percent, plugin):
         self.ui.progressBar.setFormat("%s" % plugin)
         self.ui.progressBar.setValue(percent)
