From 264510b276f18ac79ce768350c961c6915a362e9 Mon Sep 17 00:00:00 2001
From: Yann Collette <ycollette.nospam@free.fr>
Date: Thu, 4 Jun 2026 18:27:40 +0200
Subject: [PATCH] patch

---
 source/frontend/carla_shared.py | 28 ++++++++++++++--------------
 1 file changed, 14 insertions(+), 14 deletions(-)

diff --git a/source/frontend/carla_shared.py b/source/frontend/carla_shared.py
index aae07a0..65e4964 100644
--- a/source/frontend/carla_shared.py
+++ b/source/frontend/carla_shared.py
@@ -480,32 +480,32 @@ else:
     CONFIG_HOME = os.getenv("XDG_CONFIG_HOME", HOME + "/.config")
 
     DEFAULT_LADSPA_PATH  = HOME + "/.ladspa"
-    DEFAULT_LADSPA_PATH += ":/usr/lib/ladspa"
-    DEFAULT_LADSPA_PATH += ":/usr/local/lib/ladspa"
+    DEFAULT_LADSPA_PATH += ":/usr/lib64/ladspa"
+    DEFAULT_LADSPA_PATH += ":/usr/local/lib64/ladspa"
 
     DEFAULT_DSSI_PATH    = HOME + "/.dssi"
-    DEFAULT_DSSI_PATH   += ":/usr/lib/dssi"
-    DEFAULT_DSSI_PATH   += ":/usr/local/lib/dssi"
+    DEFAULT_DSSI_PATH   += ":/usr/lib64/dssi"
+    DEFAULT_DSSI_PATH   += ":/usr/local/lib64/dssi"
 
     DEFAULT_LV2_PATH     = HOME + "/.lv2"
-    DEFAULT_LV2_PATH    += ":/usr/lib/lv2"
-    DEFAULT_LV2_PATH    += ":/usr/local/lib/lv2"
+    DEFAULT_LV2_PATH    += ":/usr/lib64/lv2"
+    DEFAULT_LV2_PATH    += ":/usr/local/lib64/lv2"
 
     DEFAULT_VST2_PATH    = HOME + "/.vst"
-    DEFAULT_VST2_PATH   += ":/usr/lib/vst"
-    DEFAULT_VST2_PATH   += ":/usr/local/lib/vst"
+    DEFAULT_VST2_PATH   += ":/usr/lib64/vst"
+    DEFAULT_VST2_PATH   += ":/usr/local/lib64/vst"
 
     DEFAULT_VST2_PATH   += HOME + "/.lxvst"
-    DEFAULT_VST2_PATH   += ":/usr/lib/lxvst"
-    DEFAULT_VST2_PATH   += ":/usr/local/lib/lxvst"
+    DEFAULT_VST2_PATH   += ":/usr/lib64/lxvst"
+    DEFAULT_VST2_PATH   += ":/usr/local/lib64/lxvst"
 
     DEFAULT_VST3_PATH    = HOME + "/.vst3"
-    DEFAULT_VST3_PATH   += ":/usr/lib/vst3"
-    DEFAULT_VST3_PATH   += ":/usr/local/lib/vst3"
+    DEFAULT_VST3_PATH   += ":/usr/lib64/vst3"
+    DEFAULT_VST3_PATH   += ":/usr/local/lib64/vst3"
 
     DEFAULT_CLAP_PATH    = HOME + "/.clap"
-    DEFAULT_CLAP_PATH   += ":/usr/lib/clap"
-    DEFAULT_CLAP_PATH   += ":/usr/local/lib/clap"
+    DEFAULT_CLAP_PATH   += ":/usr/lib64/clap"
+    DEFAULT_CLAP_PATH   += ":/usr/local/lib64/clap"
 
     DEFAULT_SF2_PATH     = HOME + "/.sounds/sf2"
     DEFAULT_SF2_PATH    += ":" + HOME + "/.sounds/sf3"
-- 
2.54.0

