From 354f245de5f5c4d78526baca8f9d0f3311f6a2d1 Mon Sep 17 00:00:00 2001
From: Yann Collette <ycollette.nospam@free.fr>
Date: Tue, 7 Jun 2022 21:56:10 +0200
Subject: [PATCH] remove strip

---
 src/CMakeLists.txt | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 6402345..1d465a5 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -222,10 +222,6 @@ if (CONFIG_JACK)
     target_link_libraries (${PROJECT_NAME}_jack PRIVATE PkgConfig::LIBLO)
   endif ()
   if (UNIX AND NOT APPLE)
-    if (NOT CONFIG_DEBUG)
-      add_custom_command(TARGET ${PROJECT_NAME}_jack POST_BUILD
-        COMMAND strip ${PROJECT_NAME}_jack)
-    endif ()
     install (TARGETS ${PROJECT_NAME}_jack RUNTIME
       DESTINATION ${CMAKE_INSTALL_BINDIR})
     install (FILES images/${PROJECT_NAME}.png
-- 
2.36.1

