diff --git a/modules/juce_graphics/native/juce_linux_Fonts.cpp b/modules/juce_graphics/native/juce_linux_Fonts.cpp
index d5e9cc13e..c982a7a3b 100644
--- a/modules/juce_graphics/native/juce_linux_Fonts.cpp
+++ b/modules/juce_graphics/native/juce_linux_Fonts.cpp
@@ -222,7 +222,7 @@ Typeface::Ptr Font::getDefaultTypefaceForFont (const Font& font)
     const auto characteristics = defaultInfo.getRealFontCharacteristics (name);
     f.setTypefaceName (characteristics.name);
 
-    const auto styles = findAllTypefaceStyles (name);
+    const auto styles = findAllTypefaceStyles (characteristics.name);
 
     if (! styles.contains (font.getTypefaceStyle()))
         f.setTypefaceStyle (characteristics.style);
